bridge/service/postgres.py

```diff
@@ -18,12 +21,29 @@ class PostgresEnvironment(BaseModel):
     POSTGRES_HOST: str = "localhost"
     POSTGRES_PORT: str = "5432"
 
+    @property
+    def dsn(self) -> str:
+        return (
+            f"dbname={self.POSTGRES_DB} "
+            f"user={self.POSTGRES_USER} "
+            f"password={self.POSTGRES_PASSWORD} "
+            f"host={self.POSTGRES_HOST} "
+            f"port={self.POSTGRES_PORT}"
+        )
+
+    @property
+    def database_url(self) -> str:
+        return (
+            f"postgresql://{self.POSTGRES_USER}:{self.POSTGRES_PASSWORD}"
+            f"@{self.POSTGRES_HOST}:{self.POSTGRES_PORT}/{self.POSTGRES_DB}"
+        )
+
 
 class PostgresConfig(ContainerConfig[PostgresEnvironment]):
     image: str = "postgres:12"
     name: str = "bridge_postgres"
     ports: dict[str, int] = {"5432/tcp": 5432}
-    volumes: dict[str, Union[list[str], dict[str, str]]] = Field(
+    volumes: dict[str, list[str] | dict[str, str]] = Field(
```

Comment:
This syntax requires Python 3.10+ but we'd like to support 3.9 as well. I think adding `from __future__ import annotations` at the top of the file is probably enough to fix this!